examples/SLAM/js/common/math/plane.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Helper class for maths regarding 3D Plane
+ * Striped down and refactored to use gl-matrix from https://github.com/mrdoob/three.js/blob/dev/src/math/Plane.js
+ */
+
+import { vec3 } from 'gl-matrix';
+
+
+class Plane {
```

Comment:
I was only able to find a single use of this and the RayHit class, since that use is an intersection with an origin XY plane, I made a suggestion for some straight forward code to replace both classes with.

It seems this class appears a couple of times throughout the code base in different versions. Make sure to find all of them :)